packages/shared/src/graphql/users.ts

```diff
@@ -26,8 +26,8 @@ export type UserReadingRank = { currentRank: number };
 export type UserReadingRankData = { userReadingRank: UserReadingRank };
 
 export const USER_READING_RANK_QUERY = gql`
-  query UserReadingRank($id: ID!) {
-    userReadingRank(id: $id) {
+  query UserReadingRank($id: ID!, $timezone: String) {
```

Comment:
In the API PR, I added a comment about providing the timezone in the request.
Let's decide about it there and we'll circle back once it's set